src/armonik_cli/admin.py

```diff
@@ -18,12 +18,17 @@ def create_channel(endpoint: str,  ca: str = None, cert: str = None , key: str =
     Returns:
         grpc.Channel: gRPC channel for communication
     """
-    if ca != None and cert != None and key!= None:
-        ca_data = open(ca, 'rb').read()
-        cert_data = open(cert, 'rb').read()
-        key_data = open(key, 'rb').read()
-        credentials = grpc.ssl_channel_credentials(ca_data, key_data, cert_data)
-        return grpc.secure_channel(endpoint, credentials)
+    if ca != None:
+            with open(ca, 'rb') as ca_file:
+                ca_data = ca_file.read()
+            if cert != None and key!= None :
```

Comment:
Same as above

```suggestion
            if cert and key:
```
 and that will take care of the case if the strings are empty.